modelci/app/v1/endpoints/model.py

```diff
@@ -48,12 +49,30 @@ async def get_model_structure(id: str):  # noqa
 
 
 @router.patch('/structure/{id}')  # TODO: add response_model
-def update_model_structure_as_new(id: str, dry_run: bool = False):  # noqa
+def update_model_structure_as_new(id: str, structure, dry_run: bool = False):  # noqa
```

Comment:
how about changing the `structure` to `updated_layer`?

Reply:
How about model_operations?

Reply:
I create a temp new function `update_finetune_model_as_new`. Currently, we use this function to do model fine-tune update.